images/minio/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM minio/minio:RELEASE.2018-04-19T22-54-58Z
+
+ADD rook /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/rook"]
```

Comment:
i think `tini` should probably still be the entrypoint due to its signal handling.  the cockroach dockerfile followed the same pattern that Ceph did: https://github.com/jbw976/rook/blob/cockroachdb/images/cockroachdb/Dockerfile

Reply:
any process that runs as pid 1 has special duties in linux. See https://github.com/krallin/tini for more details.

@leseb or @BlaineEXE were recently mentioning that it might not be needed anymore due to docker doing some of the work now.  I don't know the details.

Reply:
It looks like docker supports it, but there's no way to run a container with that flag via k8s. If we want that functionality we're going to have to use tini as the entrypoint, but I'm not entirely convinced we need to use tini here. I haven't heard an explanation that justifies the requirement. This minio operator isn't spawning subprocesses unless I'm missing something.